classes/UpgradeSelfCheck.php

```diff
@@ -346,6 +344,18 @@ public function isPhpUpgradeRequired()
         return $this->phpUpgradeNoticelink = $this->checkPhpVersionNeedsUpgrade();
     }
 
+    /**
+     * @return bool
+     */
+    public function isShopVersionMatchingVersionInDatabase()
+    {
+        return version_compare(
+            Configuration::get('PS_VERSION_DB'),
+            $this->prestashopConfiguration->getPrestaShopVersion(),
+            '>='
```

Comment:
Because if for any reason the version stored in db is above the current PrestaShop, the message displayed asking to run migration files does not make sense. I want to get the message only if the version is below, and "<" is the opposite of ">=".

Reply:
I changed in favor of `==` following @AureRita checks